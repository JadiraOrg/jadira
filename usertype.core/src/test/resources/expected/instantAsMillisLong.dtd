<!ELEMENT dataset (
    INSTANTASMILLISLONG*)>

<!ELEMENT INSTANTASMILLISLONG EMPTY>
<!ATTLIST INSTANTASMILLISLONG
    ID CDATA #REQUIRED
    INSTANT CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

