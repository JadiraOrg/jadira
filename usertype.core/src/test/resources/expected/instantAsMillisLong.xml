<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "instantAsMillisLong.dtd">
<dataset>
  <INSTANTASMILLISLONG ID="0" INSTANT="1077728625760" NAME="test_0"/>
  <INSTANTASMILLISLONG ID="1" INSTANT="321581025000" NAME="test_1"/>
</dataset>